test/e2e/framework/helper/host.go

```diff
@@ -57,12 +56,12 @@ func (h *Helper) MetaDataCertificateKeyExistInHostPath(cr *cmapi.CertificateRequ
 		return err
 	}
 
-	if err := h.matchFilePerm(node, dirPath, 700); err != nil {
+	if err := h.matchFilePerm(node, dirPath, "755"); err != nil {
 		return err
 	}
 
 	metaPath := filepath.Join(dirPath, "metadata.json")
-	if err := h.matchFilePerm(node, metaPath, 600); err != nil {
+	if err := h.matchFilePerm(node, metaPath, "600"); err != nil {
```

Comment:
Wouldn't it make sense to set the file permissions on the metadata file equal to the other file permissions changed in this PR? `644`?